Header      header

float32       averaging_duration  # in seconds
MemoryUsage[] measurements
